bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -277,6 +290,17 @@ public String getRoleAttribute() {
 
     @Override
     public String getAction() {
+        List<String> supportedSubmitActions = ComponentUtils.getSupportedSubmitActions(clientBuilderFactory);
```

Comment:
Is it possible to defer the network call, this code would unnecessarily increase the error trace in log and skyline on-call would get triggered

Reply:
@rismehta 
, we can't defer the network call to submission service to fetch the supported submit actions.
This was as per our discussion and agreed upon design. In anycase , even if the api fails , we don't stop the execution or rendering at cc runtime.
This api is a basic fastly dictionary return which shouldn't fail until fastly itself is down.
cc:- @jalagari @vdua